src/components/PostList/PostList.scss

```diff
@@ -1 +1,4 @@
-//
+.PostList {
```

Comment:
Start naming classes from lowercase letters. Check other similar cases.

Reply:
@OleziO  Could you please explain then why all classes have names with capitalisation applied initially (first letter of every word in uppercase) in this and previous tasks ?  
Should I change just "PostList" class which I created? Or should I change all class names which were given by a task creator?